src/main/java/net/snowflake/ingest/connection/SecurityManager.java

```diff
@@ -110,14 +120,26 @@
    * regenerateToken - Regenerates our Token given our current user,
    *                    account and keypair
    */
-  private void regenerateToken()
+  public void regenerateToken()
   {
     //create our JWT claim object
     JwtClaims claims = new JwtClaims();
 
     //set the issuer to the fully qualified username
-    claims.setIssuer(account + "." + user);
-    LOGGER.info("Creating Token with Issuer {}", account + "." + user);
+    String publicKeyFPInJwt = null;
+    try
+    {
+      publicKeyFPInJwt = getPublicKeyFp(savePublicKey(keyPair.getPublic()));
```

Comment:
Please make sure that your code is within 80 columns on your editor. Usually when we exceed 80 columns, we break into the next line.